Makefile

```diff
@@ -1,3 +1,20 @@
 SHELL := /bin/bash
+export PYTHONPATH := $(PYTHONPATH):./lambda/src
 
--include $(shell curl -sSL -o .tardigrade-ci "https://raw.githubusercontent.com/plus3it/tardigrade-ci/master/bootstrap/Makefile.bootstrap"; echo .tardigrade-ci)
+include $(shell test -f .tardigrade-ci || curl -sSL -o .tardigrade-ci "https://raw.githubusercontent.com/plus3it/tardigrade-ci/master/bootstrap/Makefile.bootstrap"; echo .tardigrade-ci)
+
+pytest/install:
+	@ $(MAKE) install/pip/$(@D) PYPI_PKG_NAME=$(@D)
+	@ python -m pip install -r lambda/tests/requirements_dev.txt
+
+python/deps:
```

Comment:
No issues with the Makefile?  I figured I'd at least get a complaint about the target name "python/deps".

Reply:
Haha I went around and around in my head, but all my options mostly ended up sounding too nitpicky/pedantic even to me, and had their own edge cases where things break down. As is, it ought to work through the docker/run target or directly. So, good to me. Maybe as we try a few more of these, we'll start to see where it breaks down and have more ideas for improving it.

Naming-wise, I suppose we could swap `python/` for `lambda/`. But either seems fine to me.

But since you asked, this is kinda where my head was going... Making `pytest/install` an alias for `python/deps/test` and relying on the test requirements file to specify the pytest package... I'm really not sure this is any "better" than what you have though.

```
SHELL := /bin/bash
export PYTHONPATH := $(PYTHONPATH):./lambda/src

include $(shell test -f .tardigrade-ci || curl -sSL -o .tardigrade-ci "https://raw.githubusercontent.com/plus3it/tardigrade-ci/master/bootstrap/Makefile.bootstrap"; echo .tardigrade-ci)

pytest/install: python/deps/test

python/deps:
	@ echo "[$@] Installing package dependencies"
	@ python -m pip install -r lambda/src/requirements.txt

python/deps/test: python/deps
	@ echo "[$@] Installing test dependencies"
	@ python -m pip install -r lambda/tests/requirements_dev.txt

python/test: | guard/program/pytest
python/test:
	@ echo "[$@] Starting Python tests"
	pytest
	@ echo "[$@]: Tests executed!"

.PHONY: python/deps python/deps/test python/test
```

Reply:
The `requirements_dev.txt` file currently has `pytest` specified and has to specify it as the tests are dependent on it and possibly a version of it.  I think your makefile adds one more level of indirection, so I'd like to keep what's there until new needs arise.